server/db/indexTest.sql

```diff
@@ -0,0 +1,25 @@
+DROP TABLE IF EXISTS USERS,REQUESTS;
+CREATE TABLE IF NOT EXISTS USERS(
+ID UUID PRIMARY KEY NOT NULL,
+FIRST_NAME VARCHAR(80) NOT NULL,
+LAST_NAME VARCHAR(80) NOT NULL,
+EMAIL VARCHAR(80) NOT NULL UNIQUE,
+PASSWORD VARCHAR(90) NOT NULL,
```

Comment:
Don't you think `90` character spaces is too much for PASSWORD? Same with others.